deploy/helm/kuberhealthy/templates/servicemonitor.yaml

```diff
@@ -8,6 +8,9 @@ metadata:
     app: {{ template "kuberhealthy.name" . }}
     prometheus: {{ .Values.prometheus.name }}
     release: {{ .Values.prometheus.release }}
+    {{- if .Values.prometheus.serviceMonitor.additionalLabels}}
+    additionalLabels: {{ .Values.prometheus.serviceMonitor.additionalLabels}}
```

Comment:
sorry just thought of this, but with list values, i think there needs to be some extra configuration to make sure you get add every individual item... something like:
```
        {{- if .Values.deployment.args }}
        args:
{{ toYaml .Values.deployment.args | nindent 8 }}
        {{- end }}
```
https://github.com/Comcast/kuberhealthy/blob/master/deploy/helm/kuberhealthy/templates/deployment.yaml#L44-L47